### DISCUSSION TITLE
Can it be used on linux ?

### DISCUSSION THREAD

**Discussion by vricosti** [TorstenRobitzki/bluetoe#88] [Q&A] [answered]
Hi,

Would it be possible to use this library on a standard linux ? From what I can see it only works on some specific microcontroller.

**Comment by TorstenRobitzki** [comment, marked as answer]
Hi,
yes, currently the library works only with nrf51/52 out of the box. A very early version worked with a USB based HCI backend. There is an open issue to provide support HCI support (https://github.com/TorstenRobitzki/bluetoe/issues/29). So, maybe we will add that to the library in the future.

best regards,
Torsten

Torsten
